cardano-crypto-class/src/Cardano/Crypto/DSIGN/Class.hs

```diff
@@ -62,6 +64,9 @@ class ( Typeable v
 
   deriveVerKeyDSIGN :: SignKeyDSIGN v -> VerKeyDSIGN v
 
+  hashVerKeyDSIGN :: HashAlgorithm h => VerKeyDSIGN v -> Hash h (VerKeyDSIGN v)
+  hashVerKeyDSIGN = hashRaw rawSerialiseVerKeyDSIGN
```

Comment:
If we have the hash methods, do we particularly need to expose the raw serialisation? Alternatively, if we have the raw serialisation, is there especially a case for being able to override the hashing?

Reply:
This one is indeed arguable, but it's to allow for more direct, more efficient hashing for some instances (including I think, the recursive KES construction). To actually make use of this properly I think we'd also need an incremental digest computation interface, which we don't currently provide.

This may well turn out to be superfluous, in which case making it a standalone function rather than a class method later would be the sensible thing to do.